### DISCUSSION TITLE
Height comparison tour

### DISCUSSION THREAD

**Discussion by Piff91** [mytourbook/mytourbook#1079] [Q&A]
Hello, I can no longer compare my tours with the elevation function. The command is missing. How is it possible? How can I fix?

**Comment by wolfgang-ch** [comment]
Can you provide a screenshot where the command is missing

**Comment by Piff91** [comment]
![Immagine 2023-04-27 071201](https://user-images.githubusercontent.com/131846053/234765223-09c7f290-9d0d-431f-ac8e-e7ce5a732c9c.png)
![Immagine 2023-04-27 071124](https://user-images.githubusercontent.com/131846053/234765228-80c04e08-fa02-4a44-b854-a59b221ebabf.png)

**Comment by wolfgang-ch** [comment]
An elevation compare never startet in the tools menu...

![image](https://user-images.githubusercontent.com/1283445/234766471-d8266e0f-76a7-4e77-be0e-2dde1776a80d.png)

...it starts from a [reference tour](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/show-tours/tour-catalog) (the old name was tour catalog) 

![image](https://user-images.githubusercontent.com/1283445/234767349-a712820a-bb24-4cce-8c8c-6a052123e198.png)

**Comment by Piff91** [comment]
Thank you. Today i try this
________________________________
Da: Wolfgang Schramm ***@***.***>
Inviato: giovedì 27 aprile 2023 07:28
A: mytourbook/mytourbook ***@***.***>
Cc: Piff91 ***@***.***>; Author ***@***.***>
Oggetto: Re: [mytourbook/mytourbook] Height comparison tour (Discussion #1079)


An elevation compare never startet in the tools menu...

[image]<https://user-images.githubusercontent.com/1283445/234766471-d8266e0f-76a7-4e77-be0e-2dde1776a80d.png>

...it starts from a reference tour view<https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/show-tours/tour-catalog> (the old name was tour catalog)

[image]<https://user-images.githubusercontent.com/1283445/234767349-a712820a-bb24-4cce-8c8c-6a052123e198.png>

**Comment by Piff91** [comment]
There is the possibility of displaying a column with VAM? In reference paths

**Reply by wolfgang-ch** [reply]
>There is the possibility of displaying a column with VAM?

It should be possible to display a VAM column in the reference tours view, can you create a feature request for it.

**Reply by Piff91** [reply]
How can I do it?

**Reply by wolfgang-ch** [reply]
Here

![image](https://user-images.githubusercontent.com/1283445/235590773-bd6de5d9-85d7-42aa-9def-2be6db624ae4.png)
